web/src/app/details/CardActions.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { MouseEventHandler } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+import MUICardActions from '@material-ui/core/CardActions'
+import IconButton from '@material-ui/core/IconButton'
+import Tooltip from '@material-ui/core/Tooltip'
+
+interface CardActionProps {
+  primaryActions?: Array<Action | JSX.Element>
+  secondaryActions?: Array<Action | JSX.Element>
+}
+
+export type Action = {
```

Comment:
We want to use `Action` as it is used in other components as well, making it more of a struct rather than props for a specific component. We could make an `ActionProps` that maps to `Action` for sure, though I'm not getting a namespace conflict on my end- how can I reproduce this error?

Reply:
As noted below, updated with a separate `ActionProps` interface that accepts `action: Action` and `secondary?: boolean`